api/v1alpha4/conversion.go

```diff
@@ -375,3 +378,11 @@ func Convert_v1beta1_ClusterClass_To_v1alpha4_ClusterClass(in *clusterv1.Cluster
 	// ClusterClass.Status has been added in v1beta1.
 	return autoConvert_v1beta1_ClusterClass_To_v1alpha4_ClusterClass(in, out, s)
 }
+
+func Convert_v1beta1_WorkersClass_To_v1alpha4_WorkersClass(in *clusterv1.WorkersClass, out *WorkersClass, s apiconversion.Scope) error {
+	return autoConvert_v1beta1_WorkersClass_To_v1alpha4_WorkersClass(in, out, s)
```

Comment:
```suggestion
        // WorkersClass.MachinePools has been added in v1beta1.
	return autoConvert_v1beta1_WorkersClass_To_v1alpha4_WorkersClass(in, out, s)
```
nit: let's add a hint what the diff is. Similar for the func below